conductr_cli/sandbox.py

```diff
@@ -0,0 +1,115 @@
+import argcomplete
+import argparse
+import os
+from conductr_cli import sandbox_start, sandbox_stop
+
+DEFAULT_IP = os.getenv('CONDUCTR_IP', '127.0.0.1')
+
+def add_default_arguments(sub_parser):
+    sub_parser.add_argument('-i', '--ip',
+                            default=DEFAULT_IP,
+                            help='ConductR IP address. Defaults to $CONDUCTR_IP or 127.0.0.1.')
```

Comment:
We need that to expose the {ip}:{port} when starting the sandbox.

Generally I was thinking about implementing the same `default ip` logic into the `conductr-cli` as we have it in `sbt-conductr`:
- Try to get ip from `docker-machine`, else
  - Try to get ip from `boot2docker`, else
    - Try to get ip from `hostname`, else
      - 127.0.0.1

WDYT?